reference/docs-conceptual/windows-powershell/whats-new/What-s-New-in-Windows-PowerShell-50.md

```diff
@@ -179,9 +179,9 @@ PowerShell 4.0 are available in the
   logs all script blocks to the Event Tracing for Windows (ETW) event log,
   **Microsoft-Windows-PowerShell/Operational**.
 - Starting in Windows PowerShell 5.0, new Cryptographic Message Syntax cmdlets support encryption
-  and decryption of content by using the IETF standard format for cryptographically protecting
-  messages as documented by [RFC5652][73]. The Get-CmsMessage, Protect-CmsMessage, and
-  Unprotect-CmsMessage cmdlets have been added to the [Microsoft.PowerShell.Security][11] module.
+  and decryption of content using the IETF standard format for cryptographically protecting messages
+  as documented by [RFC5652][73]. The Get-CmsMessage, Protect-CmsMessage, and Unprotect-CmsMessage
```

Comment:
Aside: doesn't need to be addressed in this PR, but should these values be backticked?
```suggestion
  as documented by [RFC5652][73]. The `Get-CmsMessage`, `Protect-CmsMessage`, and `Unprotect-CmsMessage`
```